From eeaf3ad5319f7d7a37fbc3f99a6e4a7f6a478962 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Fri, 7 Oct 2022 10:51:28 +0200
Subject: [PATCH] arm64: dts: st: Add sysconf for the Cortex M33 boot on
 stm32mp251

To be able to set the Cortex-M33 vector table address, the
stm32-rproc driver needs access to this subsystem.

Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: I6c0b61f8c7a33f75835d4ec7e106832979e862af
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -2563,6 +2563,12 @@
 				status = "disabled";
 			};
 		};
+
+		a35ss_syscfg: syscon@48802000  {
+			compatible = "st,stm32mp25-a35ss-syscfg", "syscon";
+			reg = <0x48802000 0xac>;
+			status = "disabled";
+		};
 	};
 
 	mlahb: ahb@1 {
